css-ui-4/Overview.bs

```diff
@@ -2135,6 +2135,13 @@ Switching appearance: the 'appearance' property</h3>
 	giving them a <dfn export>primitive appearance</dfn>
 	where CSS can be used to restyle them.
 
+	Using ''appearance: base'' gives <a>widgets</a> a <dfn export>base
+	appearance</dfn>. <a>Widgets</a> in <span>base appearance</span> have their
+	<a>native appearance</a> suppressed where CSS can be used to restyle them,
+	just like <a>primitive appearance</a>, but also have default styles which are
+	usable and interoperable. This is unlike <a>base appearance</a> which can
```

Comment:
I think this intends to say it's unlike the `primitive appearance` (rather than `base appearance`)

Reply:
Good catch, thanks!